group_vars/all/main.yml

```diff
@@ -1,114 +1,23 @@
 ---
-env_type: development  # default to be overriden on command line
-customer: patient0  # default to be overriden on command line
+env_type: development
+customer: hello
 project_name: "{{ env_type }}-{{ customer }}"
 
-# Django
-django_configuration: Development
-
-# Ports
-django_port:    8000
-cms_port:       8081
-lms_port:       8071
-richie_port:    8061
-aliases_port:   8999
```

Comment:
This line must be in all for me unless if you think some project have no redirection nginx pod ?

Reply:
this comment and the following are not relevant as I am removing these files.